lib/grizzly/commands/command.ex

```diff
@@ -285,11 +285,42 @@ defmodule Grizzly.Commands.Command do
         {key, value}, m -> Keyword.put(m, key, value)
         {key, value, value2}, m -> Keyword.put(m, key, {value, value2})
       end)
+      |> Enum.map(fn tuple -> transform(tuple) end)
+      |> List.flatten()
     else
       meta
     end
   end
 
+  def transform({_key, nil} = tuple), do: tuple
+  def transform({:transmit_channel, _value} = tuple), do: tuple
+  def transform({:transmission_time, value}), do: {:transmit_time, value}
+  def transform({:route_changed, :changed}), do: {:route_changed, true}
+  def transform({:route_changed, :not_changed}), do: {:route_changed, false}
```

Comment:
It feels like there should be some other place to change these since they come from another part of Grizzly. This might have been my fault since I made comments early on regarding naming, but if the Z-Wave spec uses a particular name, then it seems like it would be better to stick with it for consistency. E.g., `transmission_time`.